lib/ansible/modules/system/pamd.py

```diff
@@ -380,11 +380,14 @@ def remove_module_arguments(service, old_rule, module_args):
     result = {'action': 'args_absent'}
     changed = False
     change_count = 0
+    if isinstance(module_args, str):
```

Comment:
This won't work if the received arg is a unicode - suggest using ansible.module_utils.six.string_types instead.